dashboard/app/views/levels/editors/_all.html.haml

```diff
@@ -7,10 +7,10 @@
 - content_for :body_scripts do
   %script{src: webpack_asset_path('js/levels/editors/_all.js')}
 
-= render partial: 'levels/editors/fields/bubble_choice_sublevel', locals: {f: f} unless (@level.is_a?(BubbleChoice) || @level.is_a?(CurriculumReference) || @level.is_a?(EvaluationMulti))
+= render partial: 'levels/editors/fields/video', locals: {f: f} unless (@level.is_a?(DSLDefined) || @level.is_a?(CurriculumReference))
+= render partial: 'levels/editors/fields/bubble_choice_sublevel', locals: {f: f} unless (@level.is_a?(BubbleChoice) || @level.is_a?(CurriculumReference) || @level.is_a?(EvaluationMulti) || @level.is_a?(StandaloneVideo))
```

Comment:
in the near future it should get a lot easier to tell whether a level is actually a bubble choice sublevel! so, ok to keep adding to this long list of exclusions for now :)